### PR TITLE
bump

### PR DESCRIPTION
**Pull request by bryzettler**

### CODE REVIEW

File: manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@ data:
               "table": "mobile_hotspot_infos",
               "schema": "public"
             }
+          ],
+          "crons": [
+            { "schedule": "0 0 * * *", "type": "integrity-check" }
```
Comment (deasydoesit) [inline manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml:30]: Given the potential impact, can we change these crons to working hours?

Comment (bryzettler) [reply manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml:30]: @ChewingGlass this should run at midnight right?

File: manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml [hunk 2]
```diff
@@ -51,13 +54,15 @@ spec:
...
-          image: public.ecr.aws/v0j6k5v6/account-postgres-sink-service:0.0.28
+          image: public.ecr.aws/v0j6k5v6/account-postgres-sink-service:0.0.35
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 3000
               name: http
               protocol: TCP
           env:
+            - name RUN_JOBS_AT_STARTUP
+              value: "true"
```
[the remark this answers is not in the document]
Comment (bryzettler) [reply manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml:65]: Nope, this just initiates the cron to be run at its scheduled time. The cron jobs are not running at startup without this

File: manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml (hunk 1 above)
Comment (ChewingGlass) [reply manifests/oracle-cluster/prod/helium/account-postgres-sink-service.yaml:30]: Yeah, better to run at utc midnight because of what it's correcting

**Review by ChewingGlass — APPROVED** [review]

**Pull request merged**